01-intro.Rmd

```diff
@@ -5,18 +5,24 @@ ottrpal::set_knitr_image_path()
 
 # Introduction
 
+This course is meant as a follow up to the ITCR course "Considerations for Bioinformatic Data Visualization", employing best practices while constructing data visualizations (both guided and a self-guided project) and interpreting the conclusions presented in the visualizations.
 
-## Motivation
+## Target Audience  
 
+The course is intended for cancer researchers with experience using basic informatics tools and who have an interest in data visualization.
 
-## Target Audience  
+## Motivation
 
-The course is intended for ...
+Due to the frequent use of automated algorithms to collect and process data as well as the large size of datasets, it is critical to look at collected data and produce effective summaries. Data visualization is useful for exploration, exposition, and validation of research, aiding in the understanding of underlying biology in data sets, debugging errors in informatics workflows, and identifying/avoiding sources of bias. This course is designed to guide learners through constructing effective and accessible visualizations and point to additional practice and documentation resources.
```

Comment:
I might avoid the word exposition just yet since it is jargon, or define it

Reply:
Good point. Maybe "exploration, validation, and presentation of research"?